butterknife-sample-lib/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+  }
+  dependencies {
+    classpath "${GROUP}:butterknife-gradle-plugin:${VERSION_NAME}"
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'butterknife'
```

Comment:
Got it. You might be interested in [this issue][1] I filed a while back for exposing things to processors automatically.

 [1]: http://b.android.com/187527